src/java.base/share/classes/java/net/URLPermission.java

```diff
@@ -219,11 +219,11 @@ private void init(String actions) {
         }
 
         List<String> l = normalizeMethods(methods);
-        Collections.sort(l);
+        l.sort(null);
```

Comment:
I am not opposed to this change, but I find this is slightly more ugly than Collections.sort(l); so I have to ask: Is there any noticeable benefit?

Reply:
Actually I agree with.
One more point is that List.sort(null) doesn't check at compile time that class implements `Comparable`. But Collections.sort have this check.
I will revert last commit.
@azvegint are you ok with that?

Reply:
No objections.